volume/volume.go

```diff
@@ -264,10 +264,9 @@ func ParseMountRaw(raw, volumeDriver string) (*MountPoint, error) {
 		}
 		spec.VolumeOptions.NoCopy = !copyData
 	}
-	if HasPropagation(mode) {
-		spec.BindOptions = &mounttypes.BindOptions{
-			Propagation: GetPropagation(mode),
-		}
+
+	spec.BindOptions = &mounttypes.BindOptions{
```

Comment:
We should not change the spec itself since this is what the user defined.
Instead we should update the returned mountpoint here: https://github.com/moby/moby/blob/master/volume/volume.go#L314